packages/mobile_app/lib/screens/registration/screen.dart

```diff
@@ -197,8 +200,8 @@ class RegistrationScreen extends StatelessWidget {
                 ),
               ),
               padding: EdgeInsets.symmetric(
-                horizontal: 96.0,
                 vertical: 16.0,
+                horizontal: 109.0,
```

Comment:
Looks like a magical hardcoded value that might have problems on other screen sizes, orientations and resolutions

Reply:
I get these values from figma. It will have problems in smaller screens, I guess I need to think about how implement it

Reply:
Ohhh, @rimmer it is padding for our gradient button in registration screen, it must be fixed